### PR TITLE
Feature/token factory guide

### PR DESCRIPTION
**Pull request by ramijames**

### CODE REVIEW

File: docs/guides/Building Token Factories/yourdevelopmentenv.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## 2. Setting up your Wallet
+
+Next you will need to set up your local wallet, which will house your private key and allow you to sign transactions on any network. You will have to generate keys locally. These will be used in the next step where you associate those keys with the Testnet account that you will be creating. [A quick guide on how to do this is located here](../Basics/creating-a-wallet.md).
+
+## 3. Testnet Account
+
+Lastly, you will need a Testnet account which will be the authorizing account that creates the Token Factory and mints the transactions.
+
+To set up your account, you can follow [this easy to understand guide](../Basics/create-a-testnet-account.md). You will need docker set up to create your keys, so make sure that you've completed the previous step.
```
Comment (Adam-Ultra) [inline docs/guides/Building Token Factories/yourdevelopmentenv.md:28]: the link to the faucet is missing here or in the linked doc, better give the faucet link here https://faucet.testnet.app.ultra.io/ to make it easy

Comment (Adam-Ultra) [reply docs/guides/Building Token Factories/yourdevelopmentenv.md:28]: better clarify in the accoutn creation step, one should also claim token from the faucet, which is required to create token factory. Othewise factory/token creation will fail due to insufficient UOS

File: docs/guides/Building Token Factories/creatingmetadata.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+-   A report is always generated during runtime.
+-   Reports are not generated until you close the program.
+-   Reports will help you debug your CSV files, any missing information will be posted to the report and sometimes in the terminal / console window.
+
+### Setup Folder Structure:
+
+To make it compliant with the token id card you must provide images:
+
+-   product: the principal image used when you expand the token id card
+-   square: used t o represent your asset when the token id card is collapsed
```
Comment (Adam-Ultra) [inline docs/guides/Building Token Factories/creatingmetadata.md:75]: a typo here: 
used **to** represent

File: docs/guides/Building Token Factories/creatingmetadata.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+However, the folder structure can be modified and replaced with a single folder that contains CSV files and relevant images/media files.
+
+### Setup CSV Template:
+
+1.  Use the following CSV template: [Google Sheets Metadata Tool Template](https://docs.google.com/spreadsheets/d/1Gi0iuJis-riKkyhYgMRYVnhGD6PbjvBv8U7lhRwdhNk/edit?usp=sharing) (Visit the link and go to `File -> Make a Copy` to make a copy in your own Google Drive).
+2.  Open your copied CSV template in your Google Drive and begin modifications of individual fields. (Read the first sheet for information regarding the template).
+3.  Export both `factory` and `tokens` sheets as CSV files. `File -> Download -> csv`
+4.  Rename factory file to `factory.json` and place in your root folder. (See [Folder Structure](#folder-structure) Above)
+5.  Rename token file to `tokens.json` and place in your root folder.
```
Comment (Adam-Ultra) [inline docs/guides/Building Token Factories/creatingmetadata.md:117]: should save them as factory.csv and tokens.csv, not .json

File: docs/guides/Building Token Factories/creatingmetadata.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+### Running the tool:
+
+Once the CSV files and all related images/media files are present in the folder, you can process the folder for JSON creation.
+
+1. Drag & Drop the Folder onto the Executable
+
+    a. Watch the console window for errors.
+
+    b. If errors have occurred, you can check the report generated after closing the application
+
```
Comment (Adam-Ultra) [inline docs/guides/Building Token Factories/creatingmetadata.md:154]: Is the guide for windows, mac, or linux users, or all of them. Cause drapg and drop doesn't work on ubuntu 22 when I test it. I can only use command line to do it.

File: docs/guides/Building Token Factories/firsttokenfactory.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        "lockup_time":null,
+        "conditionless_receivers":null,
+        "stat":0,
+        "meta_uris":["<YOUR UNIQ FACTORY URI>"],
+        "meta_hash":"<YOUR META HASH>",
+        "authorized_minters":[],
+        "account_minting_limit":1
+      }
+    ]'
+-p <YOUR ACCOUNT>
```
Comment (Adam-Ultra) [inline docs/guides/Building Token Factories/firsttokenfactory.md:79]: multiple line command needs to add backlash at the end. or move single quota like you did with mint token. 
image

**Review by Adam-Ultra — COMMENTED** [review]
overall looks excellent. just a few points added comments. also
- maybe better provide a template folder zip for the user to download in google drive, to save ppl some time
- metadata creation is not super clear on some concept, like what is manifest/upload.json, how they will be used, etc. but I guess we want to keep it simple at this stage, and have a detailed explanation in the variant tutorial.

File: docs/guides/Building Token Factories/creatingmetadata.md (hunk 4 above)
Comment (ramijames) [reply docs/guides/Building Token Factories/creatingmetadata.md:154]: That's good to know. I didn't test Linux.

**Review by Adam-Ultra — APPROVED** [review]

**Pull request merged**